crates/proof-of-sql/src/sql/evm_proof_plan/evm_tests.rs

```diff
@@ -0,0 +1,219 @@
+use crate::{
+    base::database::{
+        owned_table_utility::{bigint, owned_table},
+        CommitmentAccessor, OwnedTableTestAccessor,
+    },
+    proof_primitive::{
+        hyperkzg,
+        hyperkzg::{HyperKZGCommitment, HyperKZGCommitmentEvaluationProof},
+    },
+    sql::{
+        evm_proof_plan::EVMProofPlan,
+        parse::QueryExpr,
+        proof::{ProofPlan, VerifiableQueryResult},
+    },
+};
+use ark_ec::{AffineRepr, CurveGroup};
+use ark_ff::PrimeField;
+use itertools::Itertools;
+
+fn evm_verifier_with_extra_args(
+    query: &QueryExpr,
```

Comment:
Hmm it's better to use `DynProofPlan` directly since we are about to get rid of `QueryExpr` anyway